WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -132,7 +133,7 @@ private static ITransactionBroadcasterModel CreateBroadcaster(Network network)
 
 	private static IAmountProvider CreateAmountProvider()
 	{
-		return new AmountProvider(Services.Synchronizer);
+		return new AmountProvider(Services.HostedServices.Get<WasabiSynchronizer>());
```

Comment:
Theoretical question: 
Isn't it problematic that we require a non-static object inside a static function?

I tested and nothing is broken, just a question.